src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -303,6 +303,39 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
         return ResponseEntity.ok(testRun);
     }
 
+    /**
+     * GET /courses/{courseId}/exams/{examId}/student-exams/exercise-ids : Find the exercise ids for the current userexam.
+     * Currently this is used for exam notification problem statement updates as an auxiliary method.
+     *
+     * @param courseId  the course to which the student exam belongs to
+     * @param examId    the exam to which the student exam belongs to
+     * @return the ResponseEntity with status 200 (OK) and with the found exercise ids as body
```

Comment:
```suggestion
     * @return the ResponseEntity of type Long[] corresponding to an array of the found exercise ids
```
or even something along those lines. Explicitly stating it returns 200 (OK) is a bit confusing, as it doesn't always return 200. For example notFound().

Reply:
defining other types of responses(500,400,404...) would be better

Reply:
Yes, I think you are correct. 
I oriented myself on the comment from the similar method getStudentExamForSummary, but I like your reasoning. I changed it.